### ISSUE TITLE
The loss value suddenly increases

### ISSUE THREAD

**Issue by zhanlingm** [bowang-lab/BIONIC#56]
Dear Professor, hello! First of all, thank you for taking time out of your busy schedule to help me solve my doubts. Thank you very much! When I was training multiple networks using BIONIC 0.2.6 CPU version recently, the initial Loss value dropped normally, and suddenly increased to hundreds of thousands when running to 400 Epochs. I would like to ask you what is the cause of this? My guess is that there is no correlation between nodes of a certain network and other networks in the early stage of network processing, resulting in a large number of isolated nodes. When randomly sampling these nodes, neighbor nodes cannot be found, resulting in a very sparse batch network predicted. I don't know if my guess is correct, but I hope the professor can answer questions. Thank you very much!

**Comment by duncster94** [comment]
Hi - apologies for the late reply.

It sounds like your learning rate is too high. Can you copy/paste the config you're using and provide the number of networks you're integrating, as well as the number of nodes and edges in these networks?

**Comment by duncster94** [comment]
Hi, I'm going to close this due to staleness. Feel free to reopen if necessary.
